seatunnel-connectors-v2/connector-paimon/src/main/java/org/apache/seatunnel/connectors/seatunnel/paimon/utils/RowTypeConverter.java

```diff
@@ -445,9 +496,8 @@ public SeaTunnelDataType<?> visit(RowType rowType) {
 
         @Override
         protected SeaTunnelDataType defaultMethod(DataType dataType) {
```

Comment:
@Hisoka-X This `defaultMethod` method inherited from paimon seems to be difficult to obtain the fieldname.

Reply:
Please put fieldname as `UNKNOWN`.

Reply:
> Please put fieldname as `UNKNOWN`.

done.